ai4pkm_cli/commands/process_photos.py

```diff
@@ -18,25 +18,60 @@ def process_photos(self):
         import glob
         import os
 
-        # Get configurable paths
+        # Get configurable paths and settings
         if self.config:
             source_folder = self.config.get_photo_source_folder()
             destination_folder = self.config.get_photo_destination_folder()
+            albums = self.config.get_photo_albums()
+            days = self.config.get_photo_days()
         else:
             # Fallback to hardcoded paths if no config
-            source_folder = "Photostream/"
-            destination_folder = "Ingest/Photolog/Snap/"
+            source_folder = "Ingest/Photolog/Original/"
+            destination_folder = "Ingest/Photolog/Processed/"
+            albums = ["AI4PKM"]
+            days = 7
 
         self.logger.info(
             f"Processing photos from: {source_folder} -> {destination_folder}"
         )
+        self.logger.info(f"Albums to process: {albums}")
+        self.logger.info(f"Looking back {days} days")
 
+        # Export photos using AppleScript - process each album
         try:
-            subprocess.run(
-                ["osascript", "_Settings_/Tools/export_photos.applescript"], check=True
-            )
+            # Get the script path relative to the package root
+            script_path = os.path.join(os.path.dirname(__file__), "..", "scripts", "export_photos.applescript")
+            script_path = os.path.abspath(script_path)
+            
+            if not os.path.exists(script_path):
+                self.logger.error(f"AppleScript not found: {script_path}")
+                return
+            
+            self.logger.info("Exporting photos from Photos app...")
+            
+            # Process each album in the list
+            for album in albums:
+                self.logger.info(f"Processing album: {album}")
+                result = subprocess.run(
+                    ["osascript", script_path, album, source_folder, str(days)], 
+                    capture_output=True, text=True, check=True
+                )
+                
+                # Log AppleScript output for debugging
+                if result.stdout:
```

Comment:
For my test, stderr contains all the logs not stdout. Changing it to `result.stderr` worked.